### ISSUE TITLE
Make app name configurable

### ISSUE THREAD

**Issue by MLnick** [IBM/MAX-Framework#5]
This line: https://github.com/IBM/MAX-Framework/blob/master/maxfw/core/app.py#L13 

means that the app name when serving in Flask looks like: 
```
 * Serving Flask app "maxfw.core.app" (lazy loading)
 * Environment: production
...
```

Not too major, but would be nice to make the app name a configurable property.

**Comment by splovyt** [comment]
Would you want to set it equal to the API_TITLE variable or as a new variable APP_NAME?
Since we are aiming for user-friendly API titles, I personally think setting it to API_TITLE is preferable. Let me know what you think!

**Comment by MLnick** [comment]
I think `API_TITLE`  makes sense yes

**Comment by splovyt** [comment]
This was addressed in PR #6. Thanks for the suggestion!
